torchvision/datasets/sbd.py

```diff
@@ -65,11 +65,11 @@ def __init__(self,
 
         super(SBDataset, self).__init__(root, transforms)
 
-        if mode not in ("segmentation", "boundaries"):
-            raise ValueError("Argument mode should be 'segmentation' or 'boundaries'")
-
-        self.image_set = image_set
-        self.mode = mode
+        # FIXME: Is it common to not have the *.txt of the image_set in the root folder?
```

Comment:
don't think this is common (and was not even supported before), this looks good to me.